### PR TITLE
Cleanup/simplify ExpertExtender tests

### PR DESCRIPTION
**Pull request by thiemowmde**
This is a direct follow-up to #141.

### CODE REVIEW

File: tests/src/ExpertExtender/testExpertExtenderExtension.js [hunk 1]
```diff
@@ -8,15 +8,15 @@
 	valueview.tests = valueview.tests || {};
 	valueview.tests.testExpertExtenderExtension = {
 		all: function( constructor, getInstance ) {
-			this.constructor( constructor, getInstance );
-			this.destroy( constructor, getInstance );
-			this.init( getInstance );
+			var instance = getInstance();
```
Comment (adrianheine) [inline tests/src/ExpertExtender/testExpertExtenderExtension.js]: They all need their own instance. That's what the previous change was about :)

File: tests/src/ExpertExtender/ExpertExtender.Listrotator.tests.js [hunk 2]
```diff
@@ -14,11 +14,12 @@
...
-		return new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] );
-	}
-
-	testExpertExtenderExtension.all( ExpertExtender.Listrotator, getInstance );
+	testExpertExtenderExtension.all(
+		ExpertExtender.Listrotator,
+		function() {
+			return new ExpertExtender.Listrotator( '', [ { value: 'value', label: 'label' } ] );
+		}
+	);
```
Comment (adrianheine) [inline tests/src/ExpertExtender/ExpertExtender.Listrotator.tests.js]: Fine with me

File: tests/src/ExpertExtender/testExpertExtenderExtension.js (hunk 1 above)
Comment (thiemowmde) [inline tests/src/ExpertExtender/testExpertExtenderExtension.js]: Ahhh. Makes it even simpler. :-)

**Comment by tobijat** [comment]
@adrianheine merge?

File: tests/src/ExpertExtender/testExpertExtenderExtension.js [hunk 3]
```diff
@@ -8,15 +8,13 @@
 	valueview.tests = valueview.tests || {};
 	valueview.tests.testExpertExtenderExtension = {
 		all: function( constructor, getInstance ) {
-			this.constructor( constructor, getInstance );
-			this.destroy( constructor, getInstance );
-			this.init( getInstance );
+			this.constructor( constructor, getInstance() );
+			this.destroy( constructor, getInstance() );
+			this.init( getInstance() );
```
Comment (adrianheine) [inline tests/src/ExpertExtender/testExpertExtenderExtension.js]: I still think instantiation should happen in the `QUnit.test` calls. Consider that this code runs for all QUnit test runs, even when these tests are filtered out.

Comment (thiemowmde) [inline tests/src/ExpertExtender/testExpertExtenderExtension.js]: Understood, reverted.

**Comment by thiemowmde** [comment]
I reworked the patch and squished some minor style fixes in. Please tell me if this is ok for you, @adrianheine.

**Pull request merged**